src/GameServer.pas

```diff
@@ -768,7 +768,7 @@ procedure StartGame;
       begin
         Government := gDespotism;
         Money := StartMoney;
-        TaxRate := 30;
+        TaxRate := 0;
```

Comment:
I think this value was chosen for a better game play for newbies.
If you start with 30% tax rate, you won't lose buildings due to absence of gold.

A starting value of 50 gold might be good though.. Should be discussed on the forum I believe.